beangulp/__init__.py

```diff
@@ -85,9 +86,17 @@ def _identify(ctx, src):
     return ctx.identify(src)
 
 
+def _importer(imp):
```

Comment:
Please add docstring.
(google style is to insert a docstring unless the function is painfully trivial.)